lib/kitchen/directions/bake_references/v1.rb

```diff
@@ -8,7 +8,7 @@ def bake(book:, metadata_source:, numbered_title:)
       book.chapters.each do |chapter|
         chapter.search('[data-type="cite"]').each do |link|
           link.prepend(child:
-            <<~HTML
+            <<~HTML.chomp
```

Comment:
I have added methods removing new lines and white spaces (l.24) because they were causing too much additional space before the comma separator:
![image](https://user-images.githubusercontent.com/55203598/150581186-bab3a332-fd44-4926-938c-07ef08c016f0.png)